backend/src/user/user.controller.ts

```diff
@@ -47,19 +47,33 @@ export class UserController {
 	@UseGuards(AuthGuard)
 	@Get()
 	async findAll(): Promise<User[]> {
-		return this.userService.findAll();
+		return this.userService.findAll({
+			relations: { matchmaking_request: true, invite: true,
+				games_as_player_one: true, games_as_player_two: true },
+		});
 	}
 
 	@UseGuards(AuthGuard)
 	@Get(':id')
 	async findOne(@Param('id') id: number): Promise<User> {
 		return this.userService.findOne({
 			where: {
-				id: +id,
+				id: id,
 			},
+			relations: {
+				matchmaking_request: true, invite: true,
+				games_as_player_one: true, games_as_player_two: true
+			}
 		});
 	}
 
+	// @UseGuards(AuthGuard)
```

Comment:
I can imagine this is for later or something

Reply:
Nope can be removed thx!